todo_app/data/trello_items.py

```diff
@@ -31,56 +28,56 @@ def get_items():
     headers=headers,
     params=query,
     )
-  #Create a new array
+#Create a new array
   apiItems = []
-  #Convert json response to a dictionary item
+#Convert json response to a dictionary item
   aDict = json.loads(response.text)
-  #Creating a list from a dictionary item and using in the for statement
+#Creating a list from a dictionary item and using in the for statement
   for list in aDict:
     for card in list['cards']:
       apiItems.append({"id": card["id"], "title": card["name"], "status": list["name"]})
-  #returnng array of data
+#returning array of data
   return apiItems.copy()
 
-  #print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
-get_items()
-#New
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-    
-def add_item():
-  url = "https://api.trello.com/1/boards/62f2b96de0b99c00aff27ae6/cards?"
+#Add new item to Trello To Do List
+def add_item(title):
+  url = "https://api.trello.com/1/cards"
 
   headers = {
-    "Accept": "application/json"
-    }
-  load_dotenv()
-  trello_key=(os.getenv('TRELLO_KEY'))
-  trello_token=(os.getenv('TRELLO_TOKEN'))
+   "Accept": "application/json"
+  }
+
   query = {
-   'key':trello_key,
-   'token':trello_token,
-   #The ID of the list the card should be created in- so To Do
-   'idList': '62f2b96de0b99c00aff27ae6'
-#   'token': 'APIToken'
-   }
+   'idList': '62f2b96de0b99c00aff27aed',
```

Comment:
This `idList` is also hardcoded. You should create a new environment variable and get it from there

Reply:
Both this and the above are true for the `update_item` method below

Reply:
Created two new environment variables called TRELLO_TODO_LIST and TRELLO_DONE_LIST to resolve this